src/CattyTests/Bricks/MoveNStepsBrickTests.m

```diff
@@ -1,185 +0,0 @@
-/**
```

Comment:
This deprecated file is not used anymore (converted to Swift) and has been removed from the file system.